### ISSUE TITLE
JIT mode

### ISSUE THREAD

**Issue by arcanis** [elliott-wen/browservm.js#3]
The readme mentions a (closed-source, I think?) JIT mode. Do you plan to eventually open source it?

For reference I've tried running Node and it works, but very slowly (it takes a minute to run `yarn --version`, most of the time seems to be spent booting Node). One interesting bug: running `node -h` yields the following in the console:

![image](https://user-images.githubusercontent.com/1037931/105553914-9f602f80-5d06-11eb-87db-4f8fdc0d0819.png)

**Comment by elliott-wen** [comment]
Yes. Will open source it. The bug is known. It’s because the shared buffer array between workers is too small. Will fix it in next commit

Cheers
Elliott
